collectd/files/collectd_logrotate.conf

```diff
@@ -0,0 +1,4 @@
+/var/log/{{ service_name }}.log {
```

Comment:
Thanks. I checked and `copytruncate` is not necessary. Actually the collectd logfile plugin automatically recreates the `/var/log/collectd.log` file if it was removed. So we can even use the `nocreate` option and let the collectd logfile plugin create the file by itself (with the proper mode, user, etc.).